xclim/indicators/atmos/_temperature.py

```diff
@@ -437,6 +438,18 @@ class TasminTasmax(Daily2D):
     compute=indices.cold_spell_frequency,
 )
 
+cool_night_index = Tasmin(
+    identifier="cool_night_index",
+    units="degC",
+    long_name="cool night index",
+    description="Mean minimum temperature for September (northern hemisphere) or March (southern hemisphere).",
+    cell_methods="time: min within days time: mean over days",
+    comment="Original formula published in Tonietto & Carbonneau, 2004 (10.1016/j.agrformet.2003.06.001).",
```

Comment:
```suggestion
```
This information is already added in the `reference` attribute!

Reply:
Really? That's convenient. I didn't know!